mohawk/util.py

```diff
@@ -268,3 +267,37 @@ def normalize_header_attr(val):
     # Normalize like the hawk reference code.
     val = escape_header_attr(val)
     return val
+
+
+def get_bewit(resource):
```

Comment:
oh, huh, I guess you need `resource.timestamp` to be handled by `calculate_mac()` with the expiration offset